samples/tflm_ethosu/linker.ld

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Arm Limited.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(ddr4), okay)
+GROUP_START(DDR4)
+
+	SECTION_DATA_PROLOGUE(_DDR4_SECTION_NAME,,SUBALIGN(16))
+	{
+        . = ALIGN(16);
+        *(tflm_model tflm_arena tflm_input tflm_output)
```

Comment:
Just a comment here (nothing to change).

The `zephyr,memory-region` mechanism is already creating a section for you. The only difference with respect to this file is that in the section we are placing
```
*(DDR4 DDR4.*)
```
instead of:
```
*(tflm_model tflm_arena tflm_input tflm_output)
```
of course this is fine if keeping the naming is important for you.

Reply:
Think it could be an advantage to keep the label, if we want to port the sample application to another target that has a different memory layout.